src/client/styles/components/ShepContainer.scss

```diff
@@ -38,4 +38,8 @@
       margin-left: 25px;
     }
   }
+
+  h2 {
```

Comment:
I think this could go in the `SubjectHeadings/BibsList` style sheet. Also, we could use `#titles` instead of `h2` for the selector, incase we need to change the heading level again for some reason.

Reply:
Done. Also changed the font family to Milo Light to match the rest of the page